activerecord/lib/active_record/connection_adapters/sqlite3_adapter.rb

```diff
@@ -106,6 +106,7 @@ def native_database_types # :nodoc:
         binary:       { name: "blob" },
         boolean:      { name: "boolean" },
         json:         { name: "json" },
+        jsonb:        { name: "json" },
```

Comment:
Reading through https://github.com/sparklemotion/sqlite3-ruby/discussions/577, it seems like the correct column type for "jsonb" would be "blob", otherwise we're just aliasing `jsonb` to regular `json` right?